.circleci/config.yml

```diff
@@ -0,0 +1,238 @@
+          name: 'Module Build'
+          command: node build-system/pr-check/module-build.js
+  'Bundle Size':
+    executor:
+      name: amphtml-executor
+    steps:
+      - setup_vm
+      - run:
+          name: 'Bundle Size'
+          command: node build-system/pr-check/bundle-size.js
+  'Validator Tests':
+    executor:
+      name: amphtml-executor
+    steps:
+      - setup_vm
+      - run:
+          name: 'Install Protobuf'
+          command: pip3 install --user protobuf
+      - run:
+          name: 'Validator Tests'
+          command: node build-system/pr-check/validator-tests.js
+  'Visual Diff Tests':
+    executor:
+      name: amphtml-executor
+    steps:
+      - setup_vm
+      - run:
+          name: 'Visual Diff Tests'
+          command: node build-system/pr-check/visual-diff-tests.js
+  'Unminified Tests':
+    executor:
+      name: amphtml-executor
+    steps:
+      - setup_vm
+      - run:
+          name: 'Unminified Tests'
+          command: node build-system/pr-check/unminified-tests.js
+  'Nomodule Tests':
+    executor:
+      name: amphtml-executor
+    steps:
+      - setup_vm
+      - run:
+          name: 'Nomodule Tests'
+          command: node build-system/pr-check/nomodule-tests.js
+  'Module Tests':
+    executor:
+      name: amphtml-executor
+    steps:
+      - setup_vm
+      - run:
+          name: 'Module Tests'
+          command: node build-system/pr-check/module-tests.js
+  'End-to-End Tests':
+    executor:
+      name: amphtml-executor
+    steps:
+      - setup_vm
+      - run:
+          name: 'End-to-End Tests'
+          command: node build-system/pr-check/e2e-tests.js
+  'Performance Tests':
+    executor:
+      name: amphtml-executor
+    steps:
+      - setup_vm
+      - run:
+          name: 'Performance Tests'
+          command: node build-system/pr-check/performance-tests.js
+  'Experiment A Tests':
+    executor:
+      name: amphtml-executor
+    steps:
+      - setup_vm
+      - run:
+          name: 'Experiment A Tests'
+          command: node build-system/pr-check/experiment-tests.js --experiment=experimentA
+  'Experiment B Tests':
+    executor:
+      name: amphtml-executor
+    steps:
+      - setup_vm
+      - run:
+          name: 'Experiment B Tests'
+          command: node build-system/pr-check/experiment-tests.js --experiment=experimentB
+  'Experiment C Tests':
+    executor:
+      name: amphtml-executor
+    steps:
+      - setup_vm
+      - run:
+          name: 'Experiment C Tests'
+          command: node build-system/pr-check/experiment-tests.js --experiment=experimentC
+
+workflows:
+  'CircleCI PR Check':
+    jobs:
+      - 'Checks':
+          context: amphtml-context
+      - 'Unminified Build':
+          context: amphtml-context
+      - 'Nomodule Build':
+          context: amphtml-context
+      - 'Module Build':
+          context: amphtml-context
+      # TODO(rsimha): Uncomment after Travis is shut off (avoid duplicate reporting)
```

Comment:
I suggest uncommenting this here, and changing the reporting function to only work on Travis for now but still have CircleCI run everything up to the part that sends the report to the bot. wdyt?

Reply:
Good idea. Uncommented this block, and went a step further to do the reporting from CircleCI but not Travis.